openstudiocore/src/cli/openstudio/measure/ShowRunnerOutput.rb

```diff
@@ -1,3 +1,9 @@
+begin
+  require 'openstudio_measure_tester/test_helper.rb'
```

Comment:
@asparke2, @macumber. This method doesn't work here. In fact, it appears that calling `require 'openstudio/measure/ShowRunnerOutput'` in the measure test is not needed at all because when `require 'openstudio'` is called the first time, the ShowRunnerOutput is loaded. So calling it in the measure tests is returning false, because it is already loaded. The weird part is that even though it is loaded with `require 'openstudio'`, it does not load the measure_tester gem with this code.

That said, we need to figure out where to put this then. We can put it in `openstudio.rb` with no `puts` if it fails. We can then check if it is loaded in the Measure Tester gem. Thoughts?

Reply:
This is where some really weird OpenStudio init stuff happens, https://github.com/NREL/OpenStudio/blob/3e9513ccd63af80c2cadb72e4a32338c8e229bbb/openstudiocore/ruby/init_openstudio.cpp#L166

and here https://github.com/NREL/OpenStudio/blob/7865ba413ef52e8c41b8b95d6643d68eb949f1c4/openstudiocore/ruby/openstudio_rb.cpp#L71

Looks like this might not be handled consistently between the .so and CLI